Makefile

```diff
@@ -132,7 +140,7 @@ deploy: gen-manifests ## Deploy controller to the K8s cluster specified in ~/.ku
 	kubectl apply -f $(TMP_ODIR)/flotta-operator.yaml
 ifeq ($(TARGET), k8s)
 ifneq (,$(shell which minikube 2>/dev/null))
-	minikube addons enable ingress
+	minikube addons enable ingress || true
```

Comment:
perhaps we should add another possible value for `TARGET` as `kind`?
next week a PR will add cert-manager for `TARGET=k8s` as part of `make deploy` and will restructure the kustomize hierarchy to "kustomization" per cluster-type. 
The last to be merged will have to address this :-)

Reply:
Well... @arielireni let's add another TARGET for `kind` since it requires a different way to enable or deploy resources than minikube